src/static/js/sandboxPendo.js

```diff
@@ -9,7 +9,7 @@
 
 $(document).ready(() => {
   // Initialize only in Sandbox
-  if (document.location.hostname.endsWith('sandbox.nodejs.demo.rookout.com')) {
+  if (document.location.hostname.endsWith('sandbox-nodejs.rookout-demo.com')) {
```

Comment:
```suggestion
  const sandboxUrlPattern = new RegExp(/sandbox-nodejs(?:\.[a-zA-Z]+)?\.rookout-demo\.com$/);
  if (document.location.hostname.match(sandboxUrlPattern)) {
```

Reply:
@dan505512 This will work for any environment (like sandbox-nodejs.staging.rookout-demo.com)